09-big-three/SOLUTION.md

```diff
@@ -84,6 +84,36 @@ We could optimize the placement logic by utilizing a binary search sort of techn
 
 With this, we have a linear solution and we only had to make one pass.
 
+
+## Solution 3: The triple variable
```

Comment:
@mwhitman189 make sure to give yourself credit for this one by creating a link to your github user
See: https://github.com/shin10kudev/weekend-code-challenge/search?q=mwhitman189